Doc/c-api/object.rst

```diff
@@ -613,3 +613,13 @@ Object Protocol
 
    .. versionadded:: 3.14
 
+.. c:function:: int PyUnstable_IsImmortal(PyObject *obj)
+
+   This function returns non-zero if *obj* is :term:`immortal`, and zero
+   otherwise. This function cannot fail.
+
+   .. note::
+
+      Objects that are immortal in one Python version are not guarunteed to be immortal in another.
```

Comment:
There's a typo. (Also the line is too long, and I like to say `CPython` when talking about implementation details.)
```suggestion
      Objects that are immortal in one CPython version are not guaranteed to
      be immortal in another.
```